control-plane/subcommand/server-acl-init/create_or_update.go

```diff
@@ -28,6 +27,8 @@ func (c *Command) createACLPolicyRoleAndBindingRule(componentName, rules, dc, pr
 		if !primary {
 			datacenters = append(datacenters, primaryDC)
 		}
+	} else if dc != "" {
+		datacenters = append(datacenters, dc)
 	}
```

Comment:
I think we need to change this policy logic to be equivalent to what's on `main` instead: https://github.com/hashicorp/consul-k8s/blob/fbf493a092bc551e7d2a96770b1a3ef248948943/control-plane/subcommand/server-acl-init/create_or_update.go#L46-L49

Reply:
The link didn't work for me. But I think it should result in some test changes because this logic has changed. I'll take a look in another pass.

Reply:
If you look at my last commit, that's it.